### ISSUE TITLE
Accessibility - Poll can't be answered with NVDA screenreader in firefox

### ISSUE THREAD

**Issue by hassmal** [Dataport/terminfinder-frontend#304]
# Bug

Es ist nicht möglich, dem Screenreader NVDA im Firefox eine Umfrage zu beantworten. Außerdem wird der aktuelle Fokus nicht korrekt vorgelesen.

## IST-Zustand

Die gesamte Tabelle wird auf einmal vorgelesen. Danach bleibt der Screenreader still, wenn man den Fokus ändert. Außerdem sind die Elemente der Umfrage nicht mit der Tastatur bedienbar.

## Fehlermeldungen

-

## Workflow zur Reproduktion

Schritt-für-Schritt Beschreibung wie der Bug reproduziert werden kann

1. Eine Umfrage erstellen
2. Link zum Beantworten der Umfrage im Firefox öffnen
3. NVDA starten
4. In die Tabelle tabben

## SOLL-Zustand

Alle Elemente sind mit der Tastatur bedienbar und der aktuelle Fokus wird zu jeder Zeit vom Screenreader vorgelesen.

-------

# Bug

It is not possible to answer a survey to the screen reader NVDA in Firefox. In addition, the current focus is not read out correctly.

## Actual state

The entire table is read out at once. The screen reader then remains silent when the focus is changed. In addition, the elements of the survey cannot be operated with the keyboard.

## Error messages

-

## Workflow for reproduction

Step-by-step description of how the bug can be reproduced

1. create a survey
2. open the link to answer the survey in Firefox
3. start NVDA
4. tab into the table

## Desired state

All elements can be operated with the keyboard and the current focus is read out by the screen reader at all times.
